### ISSUE TITLE
Example with labelled bars

### ISSUE THREAD

**Issue by dmak** [angular-google-chart/angular-google-chart#306]
Please add the example for bar/column chart [labelled with data values](https://developers.google.com/chart/interactive/docs/gallery/barchart#labeling-bars). The critical part is: how to instantiate `google.visualization.DataView`?
```
$scope.chart.view = new google.visualization.DataView($scope.chart.data); // not found at runtime
$scope.chart.view.setColumns([
	0, 1,
	{
		calc : "stringify",
		sourceColumn : 1,
		type : "string",
		role : "annotation"
	},
	2
]);
```

**Comment by nbering** [comment]
You're welcome to make a pull request to the gh-pages branch. See https://pages.github.com/ for an explaination of GitHub pages and Jekyll.

**Comment by nbering** [comment]
Oh! Maybe I misunderstood. You want me to solve the problem for you, not write the documentation. See [my post on the api promise](http://nicholasbering.ca/angularjs/2015/04/02/google-charts-api-promise/). That should help you.

**Comment by dmak** [comment]
Thanks. I actually searching for solution (= your latest post), still cannot find one.
Example crashes with message "Error: google.visualization is undefined"… How could it be that core namespace is not loaded?
```
module.controller('InfoInputController', ['$scope', 'googleChartApiPromise', function($scope, googleChartApiPromise) {

$scope.chart = {
	type: 'ComboChart',
	data: {}
};

googleChartApiPromise.then(new function() {
	$scope.chart.view = new google.visualization.DataView($scope.chart.data); // breaks here with message "Error: google.visualization is undefined"
	$scope.chart.view.setColumns([
		0,
		{
			calc: 'stringify',
			type: 'string',
			role: 'annotation'
		}
	]);
});

}]);
```

**Comment by nbering** [comment]
That should not be possible. Can you duplicate in a live code example?

Whenever I've had people approach me with similar issues it's always been a typo or something that they didn't copy into the issue. I'm inclined to believe you're having an issue, but if we're getting a loader callback and there's no google.visualization object, then something's broken on google's end.

**Comment by dmak** [comment]
[Here is an example](https://jsfiddle.net/dma_k/pceo1jxm/). If you uncomment the `return;` statement in line 77, you should see the chart.

**Comment by nbering** [comment]
Figured it out. It's because your have the `new` keyword before the promise callback.

**Comment by dmak** [comment]
I totally agree. Here is the [corrected example](https://jsfiddle.net/dma_k/pceo1jxm/2/). However the major goal is not yet achieved: the annotations are not added. I think that `$scope.chart.data` is used to draw the graph, not `$scope.chart.view` hence column tuning is not applied. Could you please help?

**Comment by nbering** [comment]
Sorry, I didn't get back to you right away and it's been a crazy couple of months.

https://jsfiddle.net/pceo1jxm/5/

I found this upstream issue a long time ago. If you check out the hide series example on the docs site, I encountered this while build that.

The ChartWrapper we use internally doesn't actually like to take a view instance. It takes an object literal that *looks* like a view instance. When you pass a view instance, it starts getting all conflicted with their minified code. This might be something reportable as a bug to google, but I never did get around to passing it on.
